src/vellum/workflows/descriptors/utils.py

```diff
@@ -42,7 +42,7 @@ def resolve_value(
     if isinstance(value, BaseDescriptor):
         resolved_value = value.resolve(state)
         if memo is not None:
-            if value.is_sensitive:
+            if getattr(value, "_is_sensitive", False):
```

Comment:
Instead of `getattr` here, update `BaseDescriptor` to ensure it's always defined and returning `False` by default